app/src/main/res/values/strings.xml

```diff
@@ -447,8 +447,8 @@
     <string name="customize_toggle_pocket" moz:RemovedIn="108" tools:ignore="UnusedResources">Pocket</string>
     <!-- Title for the customize home screen section with Pocket. -->
     <string name="customize_toggle_pocket_2">Thought-provoking stories</string>
-    <!-- Summary for the customize home screen section with Pocket. -->
-    <string name="customize_toggle_pocket_summary">Articles powered by Pocket</string>
+    <!-- Summary for the customize home screen section with Pocket. The first parameter is product name Pocket -->
+    <string name="customize_toggle_pocket_summary">Articles powered by %s</string>
```

Comment:
```suggestion
    <string name="customize_toggle_pocket_summary_2">Articles powered by %s</string>
```
I believe you have to add this as a new string and add `moz:RemovedIn` for the old one.

Reply:
@Delphine could you double check this? I don't believe we could simply replace the existing strings, but will need to create new ones.

Reply:
I haven't exposed this yet on Pontoon so it should be OK